Formula/glibc.rb

```diff
@@ -152,12 +153,26 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add system ld search paths
+    (prefix/"etc/ld.so.conf").atomic_write(ld_so_conf)
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf"
   end
 
   def post_install
     # Install ld.so symlink.
     ln_sf lib/"ld-linux-x86-64.so.2", HOMEBREW_PREFIX/"lib/ld.so"
 
+    # Create ld.so.conf.d directories
+    (etc/"ld.so.conf.d").mkpath
+    (prefix/"etc/ld.so.conf.d").mkpath
```

Comment:
Do we need to create this? No one should be putting anything there, as it gets wiped on `brew upgrade`, so let's not add it unless we really have to.

Reply:
No, we don't. They are empty directories by far. I think I misunderstood https://github.com/Homebrew/homebrew-core/pull/109998#discussion_r970178795.